bodhi/models/models.py

```diff
@@ -1044,6 +1044,7 @@ def set_request(self, db, action, username):
         topic = u'update.request.%s' % action
         if action is UpdateRequest.unpush:
             self.unpush()
+            self.request = None
```

Comment:
I guess I am a little confused due to the two different ``unpush`` methods in play:

```bash
$ git grep unpush\(
bodhi/models/models.py:    def unpush(self, koji):
bodhi/models/models.py:                b.unpush(koji=request.koji)
bodhi/models/models.py:                up.unpush()
bodhi/models/models.py:            self.unpush()
bodhi/models/models.py:    def unpush(self):
bodhi/tests/models/test_models.py:        self.obj.builds[0].unpush(koji)
```

There's an ``unpush`` method on both the ``Build`` and on the ``Update`` class.. so I just want to make sure we have things straight and that all the cases are covered that should be.

I'll keep reading and will come back with another comment in a moment.